signer/xpi/x509.go

```diff
@@ -7,12 +7,63 @@ import (
 	"crypto/rsa"
 	"crypto/x509"
 	"crypto/x509/pkix"
+	"log"
 	"math/big"
 	"time"
 
 	"github.com/pkg/errors"
 )
 
+// every minute, add an rsa key to the cache. This will block if
+// the cache channel is already full, which is what we want anyway
+func (s *PKCS7Signer) populateRsa2048Cache() {
+	for {
+		key, err := rsa.GenerateKey(rand.Reader, 2048)
+		if err != nil {
+			log.Fatalf("xpi.populateRsa2048Cache: %v", err)
+		}
+		s.rsa2048Cache <- key
+		time.Sleep(time.Minute)
```

Comment:
The channel will block when writing if it's full, so I don't think this is needed

Reply:
I we remove this, we'll deplete the entropy when autograph starts, which could slow down other types of signing. This is a way to play nice with others.

Reply:
Oh, nice! makes sense